src/Common/src/Interop/Unix/System.Native/Interop.EnumerateInterfaceAddresses.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public unsafe struct LinkLayerAddressInfo
+        {
+            public int InterfaceIndex;
+            public fixed byte AddressBytes[8];
+            public byte NumAddressBytes;
+            private byte __pading;
```

Comment:
Thanks for explaining @pgavlin. Since we have the compiler to emit the error we should be ok.

Shouldn't an error be emitted for this struct? Looks like it's aligned at 4 bytes (I may be counting wrong).